config/default.yml

```diff
@@ -769,6 +769,10 @@ Naming/UncommunicativeMethodParamName:
     - io
     - id
     - to
+    - by
+    - 'on'
```

Comment:
Why is this in quotes?

Reply:
Because without them [it's Boolean](http://yaml.org/type/bool.html) (`true`).

[Example](https://repl.it/repls/ReadyPalatableGravity).

Reply:
This is part of Psych. `on`, `off`, `yes`, `no`, `y` and `n` are all converted too boolean.

Reply:
@Drenmi Firstly, it's part of YAML standard.

Reply:
Somehow I never noticed this. Now I'm enlightened. 😆

Reply:
@bbatsov I forgot about this, but tests failed without quoting. 😁

Reply:
Let's consider quoting all of them. Consistency is nice, and it'll prevent this question from recurring.

Reply:
@mikegee Yes, maybe you're right. I thought about this, but didn't dare.